src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,95 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Encapsulates the idea of tasks that need to be done by a certain date and time
+ */
+public class Deadline extends Task {
+
+    // constants
+    private static final String TYPE = "D";
+    private static final DateTimeFormatter DATABASE_DATE_TIME_FORMAT = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm");
+
+    // instance variables
+    private LocalDateTime deadline;
+
+    // constructors
```

Comment:
I feel that this comment can be removed since there is a Javadoc for it already. I noticed the same issue in several other places too.